sematic/resolvers/cloud_resolver.py

```diff
@@ -49,14 +54,38 @@ class CloudResolver(LocalResolver):
         will return when the entire pipeline has completed.
     """
 
-    def __init__(self, detach: bool = True, is_running_remotely: bool = False):
-        super().__init__(detach=detach)
+    def __init__(
+        self,
+        detach: bool = True,
+        _base_image_tag: str = "default",
+        _detached: bool = False,
+    ):
+        super().__init__()
+
+        # detach:
+        #   True: default, the user wants to submit a detached resolution
+        #   False: the user wants to keep resolution attached, i.e. running on their
+        #           machine
+        self._detach = detach
+        # detached:
+        #   True: we are running in a remote driver job
+        #   False: default we are running on a local user machine
+        self._detached = _detached
+
+        self._base_image_tag = _base_image_tag
```

Comment:
Please standardize the terminology between "base" (used here) and "default" (used in the bazel rule).

Reply:
They refer to different things.
`base` means base image on top of which we build images.
`default` is the default base image used for functions that don't have a custom base image.

Reply:
Done